hypervisor/arch/x86/trusty.c

```diff
@@ -419,3 +419,21 @@ bool initialize_trusty(struct vcpu *vcpu, uint64_t param)
 
 	return false;
 }
+
+void trusty_copy_dseed(uint8_t index, void *dseed)
```

Comment:
trusty_set_deseed()

Reply:
updated. all dseed will be retrieved in hob_parse.c and copy to trusty at last.